policyengine_us/variables/gov/irs/tax/federal_income/capital_gains/dwks6.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+
+class dwks6(Variable):
```

Comment:
```suggestion
class reduced_qualified_dividends_by_adjusted_investment_interest(Variable):
```

Reply:
I know it is very long but I would rather be overly accurate here than generalize too much